packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -15,15 +17,30 @@ describe('app-generator (SLOW)', function() {
   const generator = path.join(__dirname, '../../generators/app');
   const rootDir = path.join(__dirname, '../../../..');
   const sandbox = path.join(rootDir, 'sandbox/sandbox-app');
+  const sandboxDefaultVal = path.join(rootDir, 'sandbox/sandbox-default-value');
   const cwd = process.cwd();
   const appName = '@loopback/sandbox-app';
+
+  // for testing out the untildified path:
+  // use a path that contains ~ as the outdir of the app
+  const tildifySandbox = tildify(sandbox);
```

Comment:
Something to be aware of.. if this test is run outside of the current user's `$HOME` directory, then `tildifySandbox` will be identical to `sandbox`. This means the tests will pass, even if they don't actually test the `~` support. This could come up on CI systems where tests are run inside `/tmp`, for example.

I don't think anything needs to be done about it, I just wanted to point it out.

Reply:
Let' capture this information in a code comment please. If we ever run into the problem described by @rmg, then it would be great to have some help in our code.